tools/sof-dump-status.py

```diff
@@ -340,6 +341,55 @@ def _covertType(type_lst):
         print(card_info['longname'])
         exit(0)
 
+    # The kernel has changed the default firmware path when community
+    # key is used. Here we output firmware path according to kernel's
+    # match table.
+    if ret_args.get('fwpath') is True:
+        def is_community_board(board, community_boards):
+            for elem in community_boards:
+                if match_board(elem, board):
+                    return True
+            return False
+
+        def match_board(match, board):
+            for key in match["match"].keys():
+                if match["match"][key] == "":
+                    continue
+                else:
+                    if match["match"][key] != board["match"][key]:
+                        return False
+            return True
+
+        community_boards = [
+            {
+                "name": "Up Squred",
```

Comment:
@aiChaoSONG I think you mean Up Squared here.

Reply:
@stripes416 thanks, fixed the typo.